x/leveragelp/spec/incentivized_liquidation_system.md

```diff
@@ -0,0 +1,44 @@
+
+# Incentivized Liquidation System for Leverage Market
+
+## Diagram
+
+```plaintext
++------------+                  +------------+                  +-------------+                 
+|            |                  |            |                  |             |                
+| Depositors |                  | Borrowers  |                  | Liquidators |                
```

Comment:
@amityadav0 as we are managing both liquidation and SL flows, what do you think of using a different name for "Liquidators" such as Watchers or Sentinels?

Reply:
makes, sense
watchers seems good, I will add a definition below too